feconf.py

```diff
@@ -75,6 +75,12 @@
     'TextInput': 'LDAStringClassifier'
 }
 
+# The minimum number of training samples required for training a classifier.
+MIN_TOTAL_TRAINING_EXAMPLES = 50
```

Comment:
For now, this looks good. If after experimenting, we think that this needs to be increased/decreased we can always change it.